src/table.css

```diff
@@ -0,0 +1,15 @@
+.table {
```

Comment:
would recommend using [BEM](http://getbem.com/) convention for naming class names.